amps/ags/rm-community/rm-community-repo/source/java/org/alfresco/rm/rest/api/events/EventEntityResource.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * #%L
+ * Alfresco Records Management Module
+ * %%
+ * Copyright (C) 2005 - 2022 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * -
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ * -
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * -
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ * -
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+
+package org.alfresco.rm.rest.api.events;
+
+import org.alfresco.module.org_alfresco_module_rm.event.RecordsManagementEventService;
+import org.alfresco.rest.framework.WebApiDescription;
+import org.alfresco.rest.framework.resource.EntityResource;
+import org.alfresco.rest.framework.resource.actions.interfaces.EntityResourceAction;
+import org.alfresco.rest.framework.resource.parameters.CollectionWithPagingInfo;
+import org.alfresco.rest.framework.resource.parameters.Paging;
+import org.alfresco.rest.framework.resource.parameters.Parameters;
+import org.alfresco.rm.rest.api.model.EventInfo;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Event entity resource
+ *
+ * @author Swapnil Verma
+ * @since 7.4.0-M2
```

Comment:
If you find these annotations useful then I've got no problem with you adding them, but we had previously decided to stop adding them. The reasons were:
* The author:
  * is usually not a single person,
  * almost certainly isn't the most recent person to change the file
  * leads to someone being asked about code they don't know about
  * could potentially result in a false sense of "ownership"/"protectiveness"
* The release (which traditionally would be the first release containing this code (i.e. not 7.4.0-M2))
  * does not exist at the point of writing the code
  * does not match the tags in this repository
* Both pieces of information can more accurately be found using git.

Reply:
These points make sense. I added those as I thought this was a standard practice that is followed in this repo. Removed the 'author' and 'since' annotations.

Reply:
I absolutely understand why you would copy the existing practice in the repo. I just wanted to raise this discussion again. :+1: